### ISSUE TITLE
Sample weights for validation data causing val_loss to go to nan

### ISSUE THREAD

**Issue by krishnateja614** [keras-team/keras#6627]
Using keras and theano, The val_loss metric is nan when I pass sample weights for validation data. The same issue arises when passing validation data parameter to the "fit" function as well as using the model.evaluate. Train loss is not nan irrespective of sample weights parameter but only val_loss goes to nan upon passing sample weights. Don't know if the same issue exists with tensorflow backend because I can't run my model with TF backend due to low gpu memory.

**Comment by daoudChami** [comment]
I have the same issue. I think its a purely computational issue since with sample_weights = np.ones(nbItems) it works. I work on tensorflow.

**Comment by michetonu** [comment]
Any luck with this? I'm having the same issue.

**Comment by cpury** [comment]
I had the same problem and @daoudChami 's tip seems to have solved it. My `sample_weight` was a Pandas DataFrame before. Converting it to a Numpy array fixed the issue.

**Comment by cmantas** [comment]
@daoudChami 's solution seems to have worked for me too.
Weird problem though

**Comment by erbian** [comment]
confirmed issue exists with TF backend as well

**Comment by rjurney** [comment]
This happens to me with a numpy array.
